js/bookmarker.js

```diff
@@ -1,30 +1,36 @@
 /*
  * Content script to inject the dynamic bookmarking JS directly into the page.
  */
-
 (function() {
   function chromeOrBrowser() {
     return this.browser || chrome;
   }
+
   /**
    * Injected into page to redirect to the dynamic page parser
    *
    * @param {String} tenantCode
    */
-  function bookmarkPage(tenantCode, cb) {
+  function bookmarkPage(tenantCode) {
+    if (document.getElementById('bookmarkingScript')) {
```

Comment:
Remove the element if it exists on the page so after the following code we'll always end up with just a single `<script>` element on the page.